packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/views/update_pin_page.dart

```diff
@@ -0,0 +1,147 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/models.dart';
+import 'package:widget_toolkit/ui_components.dart';
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
+import 'package:widget_toolkit_pin/widget_toolkit_pin.dart';
+
+import '../../app_extensions.dart';
+import '../../lib_router/blocs/router_bloc.dart';
+import '../../lib_router/router.dart';
+import '../bloc/update_and_verify_pin_bloc.dart';
+import '../models/pin_code_arguments.dart';
+import '../services/update_and_verify_pin_code_service.dart';
+
+class UpdatePinPage extends StatefulWidget {
+  const UpdatePinPage({
+    this.pinCodeArguments = const PinCodeArguments(
+      title: '',
+      isSessionTimeout: false,
+    ),
+    super.key,
+  });
+
+  final PinCodeArguments pinCodeArguments;
+
+  @override
+  State<UpdatePinPage> createState() => _UpdatePinPageState();
+}
+
+class _UpdatePinPageState extends State<UpdatePinPage> {
+  @override
+  void initState() {
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      context.read<UpdateAndVerifyPinBlocType>().events
+        ..setPinCodeType(widget.pinCodeArguments.isSessionTimeout)
+        ..temporaryDisableBiometrics(true);
```

Comment:
You are not passing biometrics data source to `PinCodeKeyboard`.

Reply:
I have added it.